src/main.cpp

```diff
@@ -644,6 +644,7 @@ bool IsStandardTx(const CTransaction& tx, string& reason)
     return true;
 }
 
+bool CheckFinalTx(const CTransaction &tx, int nBlockHeight, int64_t nBlockTime);
```

Comment:
To avoid this declaration, would it not be possible to move the implementation of `CheckFinalTx()` (L661) above `isFinalTx()` (above L648)? IIRC we don't have function declarations in main.cpp.